pkg/wallet/wallet.go

```diff
@@ -138,9 +166,16 @@ func (w *Wallet) Path() string {
 	return w.path
 }
 
+// SetPath sets the location of the wallet on the filesystem.
+func (w *Wallet) SetPath(path string) {
+	w.path = path
+}
+
 // Save saves the wallet data. It's the internal io.ReadWriter
 // that is responsible for saving the data. This can
 // be a buffer, file, etc..
+//
+// Returns [ErrPathIsEmpty] if wallet pass is not set. See [Wallet.SetPath].
```

Comment:
OK, while we're here, can we also adjust `Save` documentation itself? To something like:
```
Save saves the wallet data to the file located at the path that was either provided via [NewWalletFromFile] constructor or via [Wallet.SetPath].
```
Not sure whether it's grammatically correct, but we need to specify this behaviour.

Reply:
Updated